www/src/components/guidelines/avatar.js

```diff
@@ -1,16 +1,18 @@
-import styled from "@emotion/styled"
+/** @jsx jsx */
+import { jsx } from "theme-ui"
 
-import { Box } from "./system"
-
-const Avatar = styled(Box)()
-
-Avatar.defaultProps = {
-  bg: `grey.10`,
-  borderRadius: 6,
-  flex: `0 0 auto`,
-  height: `avatar`,
-  lineHeight: `solid`,
-  width: `avatar`,
-}
+const Avatar = ({ ...rest }) => (
+  <div
+    sx={{
+      bg: `grey.10`,
+      borderRadius: 6,
+      flex: `0 0 auto`,
+      height: `avatar`,
+      lineHeight: `solid`,
+      width: `avatar`,
+      ...rest,
+    }}
+  ></div>
```

Comment:
Since the div doesn't have any content, we can just do a self-closing tag.

```suggestion
  />
```